test/e2e/service.go

```diff
@@ -1720,7 +1729,7 @@ metadata:
 		// with network namespaces.
 		gomega.Eventually(func() error {
 			return buildAndRunCommand(fmt.Sprintf("sudo ip netns exec client curl %s:%d/big.iso -o big.iso", svcIPforCurl, endpointHTTPPort))
-		}, 5*time.Second).Should(gomega.BeNil(), "failed to connect with external load balancer service")
+		}, 10*time.Second).Should(gomega.BeNil(), "failed to connect with external load balancer service")
```

Comment:
let me run the run E2E few more times if 10s timeout is sufficient.

Reply:
The time is getting bumped from 5 secs to 10 secs to ensure big.iso is copied over. The test was consistently failing before the time increase.

Reply:
no my question is big.iso was always copied over. and it was passing with 5seconds. so what changed in this PR that required this time bump?